llvm-bazel/llvm-project-overlay/llvm/config.bzl

```diff
@@ -41,6 +41,7 @@ linux_defines = posix_defines + [
     "HAVE_POSIX_FALLOCATE=1",
     "HAVE_SBRK=1",
     "HAVE_STRUCT_STAT_ST_MTIM_TV_NSEC=1",
+    "LLVM_USE_PERF=1",
```

Comment:
I'm nervous about even doing this in upstream. I think this really needs to be detected on the system by Bazel before we enable it.

I'd just keep all of this stuff disabled until the build files land fully, and then work on enabling it in the upstream build because I think we'll need to spend some time getting the conditions correct.

Reply:
Done